### ISSUE TITLE
'Meet the team' page - should we add descriptions?

### ISSUE THREAD

**Issue by hdrdavies** [dwyl/dwyl-site#87]
I like what JBi have done with their 'meet the team' page. Similar to the idea @nelsonic had around a short bio and interesting facts. :+1: 

![profile](https://cloud.githubusercontent.com/assets/13098199/12721408/9cad95f0-c8f7-11e5-812c-0e28ce59dc86.png)

**Comment by iteles** [comment]
👍🏻 This might be a little too wordy for a first iteration. I prefer the simplicity of something we can get up on the site quickly without looking incomplete for now

**Comment by hdrdavies** [comment]
@iteles Agreed. More in line with examples in #78, especially in terms of length

**Comment by hdrdavies** [comment]
#96 is lacking a page for this, will be completed next week :+1:

**Comment by hdrdavies** [comment]
@iteles could you put the Node Girls screenshot here please? :smile:

**Comment by iteles** [comment]
<img width="1056" alt="screen shot 2016-02-09 at 14 38 55" src="https://cloud.githubusercontent.com/assets/4185328/12919237/19d4ff84-cf3b-11e5-95ab-f79a1076ecde.png">

If you want to run it just pull down the `gh-pages` branch and `open index.html` (no server required yet)

**Comment by hdrdavies** [comment]
@iteles @nelsonic We like the Node Girls team page and we're conscious that this doesn't have a short description for each person - would you prefer a short intro for each person or not?

Perhaps we should go for no short bio for now as it will be quicker to build; if this is the case would you mind telling us who we should put up for now?

**Comment by iteles** [comment]
:+1: on the no bio for the first iteration

dwyl at the moment is Nelson and I, @heron2014 @SimonLab @jrans @des-des @rjmk @minaorangina   @nikhilaravi @jackcarlisle @Conorc1000 @naazy @msmichellegar and you guys!

**Comment by hdrdavies** [comment]
:+1: :+1: :+1:

**Comment by markwilliamfirth** [comment]
Closing in favour of #283 #288
